proof-producer/tests/bin/proof-producer/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+add_custom_target(tests_prover)
+
+# Add test for OutputArtifacts library
+# .cpp file must have the name of target
+function(add_prover_test target)
+    add_executable(${target} ${target}.cpp)
+
+    target_link_libraries(${target} PRIVATE
+        GTest::gtest GTest::gtest_main
+        proof_generatorAssigner
+        proof_generatorPreset
+        proof_generatorOutputArtifacts
+    )
+
+    target_include_directories(${target} PRIVATE
+        ${CMAKE_CURRENT_SOURCE_DIR}/../../../bin/proof-producer/include
```

Comment:
I suggest to export includes from the proof-producer executable cmake via interface library instead of hardcoding relative paths

Reply:
done